database/sql/V021__store_full_quotes.sql

```diff
@@ -0,0 +1,27 @@
+-- This migration step will change how we store min-fee measurements to store
+-- full quote data that can later be attached to an order.
+--
+-- This involves adding new columns for buy and sell amounts in the quotes as
+-- well as renaming the tables so its more apparent what they are for.
+-- Additionally, for the quotes table, we add a `bigserial` ID column for
+-- generating unique quote IDs for each order.
+
+ALTER TABLE min_fee_measurements RENAME TO quotes;
+ALTER INDEX min_fee_measurements_token_expiration RENAME TO quotes_token_expiration;
```

Comment:
Maybe we should create an index of `(sell_token, buy_token, expiration)` instead of just `(sell_token, expiration)`? I imagine this may be left over from when we had fee measurements per token and not trade.

Reply:
That would make sense to me.